standards/development-principles.md

```diff
@@ -0,0 +1,93 @@
+---
+category: Development Principles
+expires: 2019-03-31
+---
+
+# Development Principles
+
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within the MOJ) have no way of knowing for sure if they can use it, or
+how.
```

Comment:
This section is already included and expanded on by the [existing page on licensing](https://github.com/ministryofjustice/technical-guidance/blob/master/standards/licencing-software-or-code.md).

Reply:
Well spotted, I had copied an existing page and forgot to remove this. As a result of it I had forgotten to port over the openning paragraph which I have now added.